Packs/Twitter/Integrations/Twitterv2/Twitterv2_test.py

```diff
@@ -0,0 +1,176 @@
+import json
+import io
+import freezegun
+import pytest
+from CommonServerPython import DemistoException
+
+
+expected_hr_twitter_user_get = '### Twitter user get results:'\
+                               '\n|Name|User name|Created At|Description|Followers Count|Tweet Count|Verified|\n'\
+                               '|---|---|---|---|---|---|---|\n|'\
+                               ' Some_name | Some_user_name | 2024-11-09T09:55:45.000Z | Some_description | 12 | 15 | True |\n'\
+
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+test_data = util_load_json('test_data/test_data.json')
+
+
+def side_effect_twitter_tweet_search(method, url_suffix, headers, params, ok_codes):
+    if int(params.get('max_results')) and (
+        int(params.get('max_results')) < 10 or int(params.get('max_results')) > 100
+    ):
+        raise DemistoException(
+            message=f"The max_results query parameter value [{params.get('max_results')}] is not between 10 and 100",
+            res={
+                "errors": [
+                    {
+                        "parameters": {"max_results": f"[{params.get('max_results')}]"},
+                        "message": f"The max_results query parameter value [{params.get('max_results')}] "
+                        "is not between 10 and 100",
+                    }
+                ],
+                "title": "Invalid Request",
+                "detail": "One or more parameters to your request was invalid.",
+                "type": "invalid-request",
+            }
+        )
+    if params.get('start_time') and params.get('end_time'):
+        return test_data["search_tweets_response_dates"]
+    return test_data["search_tweets_response"]
+
+
+@pytest.mark.parametrize('limit, start_time, end_time ,expected_output\
+                         ,expected_human_readable, expected_raw', [
+                         (10, '', '', test_data['search_tweet_output'],
+                          test_data['search_tweets_human_readable'], test_data['search_tweets_response']),
+                         (3, '', '', test_data['search_tweet_output'],
```

Comment:
This use case should throw an error. no?
The limit is not right

Reply:
yes it throws an error in "side_effect_twitter_tweet_search" and check it in line number 88  
`    if limit < 10 or limit > 100:
        with pytest.raises(DemistoException):
            result = twitter_tweet_search_command(client, args)`